### PR TITLE
Add OffscreenCanvas extern

### PR DESCRIPTION
**Pull request by dsanders11**

### CODE REVIEW

File: externs/browser/html5.js [hunk 1]
```diff
@@ -105,6 +105,40 @@ HTMLCanvasElement.prototype.getContext = function(contextId, opt_args) {};
...
+ */
+OffscreenCanvas.prototype.getContext = function(contextId, opt_args) {};
+
+/**
+ * @return {!ImageBitmap}
+ */
+OffscreenCanvas.prototype.transferToImageBitmap = function() {};
+
+/**
+ * @param {!Object=} opt_options
```
Comment (tjgq) [inline externs/browser/html5.js]: It seems reasonable to provide a type for opt_options here:

`@param {{type: (string|undefined), quality: (number|undefined)}=} opt_options`

Comment (tjgq) [reply externs/browser/html5.js]: I ended up making the change myself in the version I imported for internal review.

**Comment by EatingW** [comment]
Internal Google issue b/120548190 created to keep track of this pull request.

**Comment by tjgq** [comment]
FYI: this PR got submitted, but had to be rolled back shortly afterwards because it broke an internal project. I'm working on getting it submitted again - there's no need to send a new PR.

**Pull request closed without merging**